FluidNC/src/Maslow/Maslow.cpp

```diff
@@ -327,8 +428,8 @@ bool Maslow_::take_measurement_avg_with_check(int waypoint) {
           run = 0;
 
           //check if all measurements are within 1mm of each other
-          static double maxDeviation[4] = { 0 };
-          static double maxDeviationAbs = 0;
+          double maxDeviation[4] = { 0 };
```

Comment:
Don't these need to be static to hold their value between calls?

Reply:
Exactly opposite. Otherwise it takes one bad measurement to kill the calibration

Reply:
Gotcha, good catch